aixplain/factories/dataset_factory.py

```diff
@@ -39,7 +39,7 @@
 from aixplain.enums.language import Language
```

Comment:
Please check which Nur's requirements we will be able to fulfill:

1. You will support all files on root and arrange them in one col in a dataset
2. You will support all folders on root and treat them as columns in the dataset
3. You will not support fixed files and folders on root. How do you deal with this? Exception or msg to the user?
4. You will not support embedded folders in folders. How do you deal with this? Exception or msg to the user?